### DISCUSSION TITLE
Possible to Add Multiple wLED Instances Simultaneously?

### DISCUSSION THREAD

**Discussion by ItsMrJP** [awawa-dev/HyperHDR#40] [Q&A]
Have been trying to figure out how to get a second profile/wLED light working, but every time I end up wiping out the first wLED's settings. I am on version 14 currently. Any advise would be greatly appreciated! Would love to add the philips hue lights + all my wLEDs if its possible.

**Comment by awawa-dev** [comment]
Hi
Should work as I tested it: on default instance I setup first WLED with IP1. Then created second instance (configuration->general). Switch to second instance (top bar), select WLED and entered IP2. Similar you can add Philips Hue instance.
If it is not working try to delete your configuration from home folder and start from scratch because there is possibility that it's broken.
